tests/web/test_basic_views.py

```diff
@@ -13,15 +13,22 @@ def test_get_transaction_endpoint(b, client, user_vk):
     input_tx = b.get_owned_ids(user_vk).pop()
     tx = b.get_transaction(input_tx['txid'])
     res = client.get(TX_ENDPOINT + input_tx['txid'])
+    assert res.status_code == 200
     assert tx == res.json
+
+    res = client.get(TX_ENDPOINT + input_tx['txid'] + '/')
```

Comment:
Just a thought: what if we had parameterized tests for each of these API tests that made sure both trailing and no-trailing slash endpoints work?